### DISCUSSION TITLE
Welcome to ArchiveBox Discussions!

### DISCUSSION THREAD

**Discussion by pirate** [ArchiveBox/ArchiveBox#576] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BlipRanger** [comment]
Hello! First off, thanks for the awesome tool! I was only introduced to it a few days ago and I'm really looking forward to taking it for a spin. I personally archive youtube videos for the most part but I'm looking for a new workflow to bookmark + archive important pages and I think that ArchiveBox will be at the crux of that. I'm thinking of using something like Wallabag on the front and have ArchiveBox pull from an RSS feed so that all the archiving happens in the background (unless you think there is something quicker/simpler). The closer I can get to a single click archival, the better. How's v0.5.0 looking? I'm really excited for the full-text search you guys cooked up. Thanks again!

**Reply by ekiel** [reply]
My favorite way to archive is simply append a text file of links and then I have a cron job run the archive overnight.
